python_orocos_kdl/PyKDL/std_string.sip

```diff
@@ -49,18 +49,17 @@
      }
      if (PyUnicode_Check(sipPy)) {
         PyObject* s = PyUnicode_AsEncodedString(sipPy, "UTF-8", "");
-        *sipCppPtr = new std::string(PyUnicode_AS_DATA(s));
+        *sipCppPtr = new std::string(PyBytes_AS_STRING(s));
         Py_DECREF(s);
         return 1;
      }
 #if PY_MAJOR_VERSION < 3
-     if (PyString_Check(sipPy)) {
-        *sipCppPtr = new std::string(PyString_AS_STRING(sipPy));
```

Comment:
As above, this is within `#if PY_MAJOR_VERSION < 3` block, so it can be left untouched